packages/wdio-browser-runner/package.json

```diff
@@ -50,14 +50,14 @@
     "webdriverio": "8.1.0",
     "ws": "^8.10.0"
   },
-  "scripts": {
-    "prepare": "rimraf node_modules/@wdio/config node_modules/@wdio/repl node_modules/@wdio/utils"
-  },
   "publishConfig": {
     "access": "public"
   },
   "devDependencies": {
     "@types/ws": "^8.5.3",
-    "@wdio/runner": "8.1.0"
+    "@wdio/config": "8.1.0",
+    "@wdio/repl": "8.1.0",
+    "@wdio/runner": "8.1.0",
+    "devtools": "8.1.0"
```

Comment:
Why is this needed when we have them in the `dependencies` section?

Reply:
config, repl, runner, devtools aren't listed in dependencies for `wdio-browser-runner`. they are dependencies of:
```
    "@wdio/local-runner": "8.1.0",
    "@wdio/logger": "8.1.0",
    "@wdio/mocha-framework": "8.1.0",
    "@wdio/protocols": "8.1.0",
    "@wdio/types": "8.1.0",
    "@wdio/utils": "8.1.0",
    "webdriver": "8.1.0",
    "webdriverio": "8.1.0",
```
and apparently are installed at the top level of `wdio-browser-runner`'s node_modules (npm? lerna?), but not replaced with a symlink by lerna. they are also installed in the sub level `node_modules` (of each dependency) which is symlinked.